recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ outputs:
       number: {{ number }}
       run_exports:
         - {{ pin_subpackage("ucx", max_pin="x.x") }}
+      ignore_run_exports:
+        - rdma-core
```

Comment:
IIUC this is because we want a more lax pinning than `rdma-core` uses. Wonder if we should just use a more lax pinning in `rdma-core` ( https://github.com/conda-forge/rdma-core-feedstock/issues/12 ). If we did then, we wouldn't need `ignore_run_exports`

Not a blocker necessarily. It would just be preferable to use packages' `run_exports` when possible

Reply:
I agree, let's finish the discussion in https://github.com/conda-forge/rdma-core-feedstock/issues/12 . I'm definitely a "maybe" there, as long as we can prevent falling into same types of pitfalls we have with PyPI as I described in https://github.com/conda-forge/rdma-core-feedstock/issues/12#issuecomment-1829917563 , I'd be ok with keeping no `max_pin`.